[DOCS-11337] Document new 'Fail on incomplete certificate chain' advanced option for Synthetic SSL tests (#30135)
DataDog/documentation: content/en/synthetics/api_tests/ssl_tests.md

   1. Add **Advanced Options** (optional) to your test:
      * **Accept self-signed certificates**: Bypasses any server error related to a self-signed certificate.
      * **Fail on revoked certificate in stapled OCSP**: Fail the test if the certificate is labeled as revoked by the OCSP stapling.

>       * **Fail on incomplete certificate chain**: Disables AIA fetching so the test fails if an intermediate certificate is not present in the chain provided by the server.

Reviewer: ```suggestion
      * **Fail on incomplete certificate chain**: Disables AIA fetching, causing the test to fail if any intermediate certificates are missing from the server's certificate chain.
```
